training/distributed_training/pytorch/data_parallel/maskrcnn/train_pytorch_smdataparallel_maskrcnn.py

```diff
@@ -1,180 +1,150 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
Why is the license changing? Is this intentional?

Reply:
yes, intention is to keep it consistent with [upstream license ](https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/Segmentation/MaskRCNN/pytorch/tools/train_net.py#L1)

Reply:
Thanks - is this file an identical copy of the Facebook file, or is it modified by Amazon?

Reply:
It was modified in our forked repo, and license was [kept as-is](https://github.com/HerringForks/DeepLearningExamples/blob/master/PyTorch/Segmentation/MaskRCNN/pytorch/tools/train_net.py#L1). Do we need to update both to add Amazon copyright?